pkg/relayerproxy/interface.go

```diff
@@ -0,0 +1,26 @@
+package relayerproxy
```

Comment:
I noticed that the package name is `relayerproxy`.

Not a blocker but curious to understand the intuition of this instead of 

`pkg/relayer/proxy/interface.go`?

Reply:
Indeed `pkg/relayer/proxy/` is better. I first had `pkg/proxy/` and didn't want to have `pkg/proxy/proxy`.

Will update the code and PR naming :+1:

Reply:
This would also mean that the `Miner` module would live in `pkg/relayer/miner` right?

Reply:
Yup!